python/python/Ice/PythonDispatcher.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) ZeroC, Inc.
+
+import inspect
+import sys
+
+def is_future(obj):
```

Comment:
We previously did:

```
if isinstance(result, Future) or callable(
                getattr(result, "add_done_callback", None)
            )
```

But the code only uses `add_done_callback` which is also provided by Ice.Future. Doesn't make much sense to have separate checks.